PWGHF/HFC/TableProducer/correlatorDplusHadrons.cxx

```diff
@@ -77,7 +77,7 @@ struct HfCorrelatorDplusHadronsDplusSelection {
   Partition<soa::Join<aod::HfCand3Prong, aod::HfSelDplusToPiKPi, aod::HfCand3ProngMcRec>> selectedDplusCandidatesMc = aod::hf_sel_candidate_dplus::isSelDplusToPiKPi >= selectionFlagDplus;
 
   void processDplusSelectionData(aod::Collision const& collision,
-                                 soa::Join<aod::HfCand3Prong, aod::HfSelDplusToPiKPi> const&)
+                                 soa::Join<aod::HfCand3Prong, aod::HfSelDplusToPiKPi> const& candidates)
```

Comment:
Do not revert fixes.

Reply:
What should I do here? I didn't change anything here.

Reply:
Fixed again in https://github.com/AliceO2Group/O2Physics/pull/5757

Reply:
Correct